src/types/config.ts

```diff
@@ -0,0 +1,15 @@
+import { ConnectionOptions } from 'rethinkdb';
+
+export interface ConfigType {
+    token: string;
+    prefix: string;
+    rethink: ConnectionOptions;
+    hypixel: string;
+    aiurl: string;
+    secret: string;
+    redirect_uri: string;
+    domain: string;
+    cors: string;
+    aitoken: string;
+    safeBrowsing: string;
+}
```

Comment:
is this actually used anywhere in the code? i removed it when we moved to .env

Reply:
IDK why it got craetedf